src/client/browserContext.ts

```diff
@@ -59,11 +65,23 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel,
     super(parent, type, guid, initializer);
     if (parent instanceof Browser)
       this._browser = parent;
+    this._isChromium = this._browser?._name === 'chromium';
```

Comment:
I think we should have `initializer.isChromium` instead, which is also set for Electron and Android.

Reply:
i'll follow up